lib/Twig/TemplateWrapper.php

```diff
@@ -99,6 +99,8 @@ public function renderBlock($name, $context = array())
      *
      * @param string $name    The block name to render
      * @param array  $context An array of parameters to pass to the template
+     *
+     * @return string
```

Comment:
Same here, the `displayBlock` method does not return anything. Here the code is just wrong and should be fixed instead.